src/sr_od/app/div_uni/operations/choose_oe_file.py

```
@@ -24,8 +24,8 @@ def check_screen(self) -> OperationRoundResult:
         识别当前所在的画面
         :return:
         """
-        if self.num < 1 or self.num > 4:
-            return self.round_fail('存档编号只能是1~4')
+        if self.num > 4:
+            return self.round_fail('存档编号只能是0~4，0代表使用默认档案')
```

Ruff (0.15.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)